package.json

```diff
@@ -166,7 +188,8 @@
     "postinstall": "node node_modules/vscode/bin/install",
     "just-test": "node node_modules/vscode/bin/test",
     "test-compile": "tsc -p ./ && npm run webpack",
-    "test": "npm run test-compile && node node_modules/vscode/bin/test"
+    "test": "npm run test-compile && node node_modules/vscode/bin/test",
+    "test-live-share": "npm run test-compile && node src/live-share/test/live-share-test"
```

Comment:
For the aforementioned reasons, I've created the separate `npm` script for running Live Share related tests.

Reply:
i like that this is a separate command. but i also think peacock should have a test script that runs them all.

Reply:
Sounds good.

Reply:
can you make that change so all the tests run in CI?